dart/packages/fury/lib/src/serializer/array/num_array_serializer.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:typed_data';
+import 'package:fury/src/deserializer_pack.dart';
+import 'package:fury/src/memory/byte_reader.dart';
+import 'package:fury/src/memory/byte_writer.dart';
+import 'package:fury/src/serializer/array/array_serializer.dart';
+import 'package:fury/src/serializer_pack.dart';
+import 'package:fury/src/util/math_checker.dart';
+
+abstract base class NumArraySerializer<T extends num> extends ArraySerializer<T> {
```

Comment:
```suggestion
abstract base class NumericArraySerializer<T extends num> extends ArraySerializer<T> {
```

Reply:
change file name too. But I think we can move this class into array_serializer.dart.

Does dart allow declare multiple class in one file